### PR TITLE
Update/about usrse

### PR DESCRIPTION
**Pull request by cosden**
This PR adds two "About US-RSE" pieces:
1. Text to the front page "What is US-RSE?" box that captures our organization mission in a (hopefully) succinct way. 
2. Significantly updates text on the "About US-RSE" page to include a more organizationally focused section at the top, minor updates to the Research Software piece that was there, and then adds a historical piece at the end.

### CODE REVIEW

**Comment by cosden** [comment]
This would close #16

File: pages/about/about.md [hunk 1]
```diff
@@ -7,32 +7,40 @@ menubar: about
...
+
+The US-RSE Association is committed to providing an inclusive environment with equitable treatment for all and to promoting and encouraging diversity throughout the RSE community in the US.
+We encourage you to read our full [DEI mission statement](https://us-rse.org/dei-mission/).
+
+[Membership]({{ site.baseurl }}/join/) in US-RSE is easy and free. We welcome all, regardless of geographic location or affiliation, who support our organizational mission.
+Our members come from all types of institutions/organizations and all types of roles including those who identify as RSEs, those interested in a career as an RSE, and those who may not be RSEs but consider themselves RSE “allies” or manage individuals in RSE roles.
+
+## Research Software
+The increasing use of digital technologies across research communities has gone hand in hand with a strong growth and reliance on software written or customized to solve research problems: research software.
+This in turn is driving dialog around opportunities to:
```
Comment (mtbc) [inline pages/about/about.md:21]:
```suggestion
In turn, this is driving dialog around opportunities to:
```
Absolutely feel free to disagree and ignore.

File: pages/about/about.md [hunk 2]
```diff
@@ -7,32 +7,40 @@ menubar: about
...
-universities, laboratories, knowledge institutes, companies and other
-enterprises. This site is part of that effort.
+[Netherlands](http://nl-rse.org) and elsewhere are bringing together the community of people writing and contributing to research software at the national and international level.
+In the US this encompasses universities, laboratories, knowledge institutes, companies and other enterprises.
+US-RSE is part of that effort.
+
+## History of The US-RSE Association
+The US-RSE Association can trace its roots back to two activities in the winter of 2017-2018.
+The first was an [international survey of RSEs](https://www.software.ac.uk/blog/2018-03-12-what-do-we-know-about-rses-results-our-international-surveys).
+There was a final question on that survey “Would you be interested in helping to form a community of RSEs in the US?”
```
Comment (mtbc) [inline pages/about/about.md:37]:
```suggestion
There was a final question on that survey, “Would you be interested in helping to form a community of RSEs in the US?”
```
Again, this really is just opinion. I also wondered about a colon.

File: pages/about/about.md [hunk 3]
```diff
@@ -7,32 +7,40 @@ menubar: about
...
+Out of nearly 200 respondents, about a dozen people answered 'yes' and started talking.
+Shortly after that, the UK RSE Association sponsored the first [International RSE Leaders Workshop](https://society-rse.org/events/international-leaders-workshop/) - a meeting designed to bring together RSE team leads from across the world to connect and share experiences.
+Five US representatives traveled to London to participate in the workshop and left energized and committed to developing an RSE community in the US.
+The first US-RSE website and slack channel were formed days later.
+Progress was slow to start but in the ensuing months and years a formal organization emerged.
+For more and to see what’s been happening recently we invite you to read our [newsletters]({{ site.baseurl }}/newsletters/), read more about our [mission]({{ site.baseurl }}/mission/), and [join]({{ site.baseurl }}/join/) the US-RSE community.
```
Comment (mtbc) [inline pages/about/about.md:43]:
```suggestion
For more, and to see what’s been happening recently, we invite you to read our [newsletters]({{ site.baseurl }}/newsletters/), read more about our [mission]({{ site.baseurl }}/mission/), and [join]({{ site.baseurl }}/join/) the US-RSE community.
```
Again, this is but a very gentle suggestion.

File: pages/about/about.md (hunk 1 above)
Comment (cosden) [reply pages/about/about.md:21]: I'm on the fence with this. I like the emphasis starting with This makes. But we may be missing some commas. Would:

"This, in turn, ..." be better?

I'm no English major, so I'm often reluctant to have a strong opinion on these matters!

And thanks for the careful review, @mtbc!

Comment (mtbc) [reply pages/about/about.md:21]: Yes, I too think that would be better but am also no English major!

Comment (cosden) [reply pages/about/about.md]: Updated!

**Review by vsoch — APPROVED** [review]

**Pull request merged**